source/index.html.erb

```diff
@@ -11,18 +11,24 @@ title: Ralph's Launch Cards
   </ul>
 </details>
 
-<div class="card">
+<div id="card" class="card">
+
```

Comment:
There's some extra whitespace in these templates.

Reply:
I adjusted spacing; we can review best practices again for spacing HTML; I know SASS expectations, just need a few more notes on HTML layout.